docs/user/nifi.rst

```diff
@@ -192,21 +245,60 @@ are available:
     For example, setting ``geomesa.sft.name`` to a non-recurring value could end up creating a new schema for each
     flow file, potentially crashing your database by creating too many tables.
 
-Feature Writer Caching
-^^^^^^^^^^^^^^^^^^^^^^
+Record Input Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Feature writer caching can be used to improve the throughput of processing many small flow files. Instead of a new
-feature writer being created for each flow file, writers are cached and re-used between operations. If a writer is
-idle for the configured timeout, then it will be flushed to the data store and closed.
+Record-based processors accept the following configuration parameters for specifying the input source. Each
+datastore-specific processor also has additional parameters for connecting to the datastore, detailed in the
+following sections.
 
-Note that if feature writer caching is enabled, features that are processed may not show up in the data store
-immediately. In addition, any features that have been processed but not flushed may be lost if NiFi shuts down
-unexpectedly. To ensure data is properly flushed, stop the processor before shutting down NiFi.
++-----------------------------------+-----------------------------------------------------------------------------------------------------+
+| Property                          | Description                                                                                         |
++===================================+=====================================================================================================+
+| ``Record reader``                 | The Record Reader to use for deserializing the incoming data                                        |
++-----------------------------------+-----------------------------------------------------------------------------------------------------+
+| ``Feature type name``             | Name to use for the simple feature type schema. If not specified, will use the name                 |
+|                                   | from the record schema                                                                              |
++-----------------------------------+-----------------------------------------------------------------------------------------------------+
+| ``Feature ID column``             | Column that will be used as the feature ID. If not specified, a random ID will be used              |
++-----------------------------------+-----------------------------------------------------------------------------------------------------+
+| ``Geometry columns``              | Column(s) that will be deserialized as geometries, and their type, as a                             |
+|                                   | SimpleFeatureType specification string (e.g. ``the_geom:Point``). A '*' can be used to                   |
```

Comment:
Since you say "can" be used, what are the other options? 

If one uses just the column names, what happens?

Reply:
it would be the same as not specifying a default geom in a simple feature type... i think it uses the first one in that case? i can check and add a clarification if you think it's worthwhile

Reply:
Eh, I think it is worth us being clear.  Otherwise, future us will have to read the code;).